openapi/paths/reports@revenue-audit.yaml

```
@@ -0,0 +1,51 @@
+servers:
+  - url: 'https://api-sandbox.rebilly.com/experimental/organizations/{organizationId}'
+    description: Sandbox server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+  - url: 'https://api.rebilly.com/experimental/organizations/{organizationId}'
+    description: Live server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+get:
+  x-products:
+    - Reports
+  tags:
+    - Reports
+  summary: Retrieve a revenue audit report
+  operationId: GetRevenueAuditReport
+  x-sdk-operation-name: getRevenueAudit
+  description: Retrieve a revenue audit report.
+  security:
+    - SecretApiKey: []
+    - JWT: []
+    - ApplicationJWT: []
+  parameters:
+    - $ref: ../components/parameters/collectionFilter.yaml
+    - $ref: ../components/parameters/collectionSort.yaml
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+  responses:
+    '200':
+      description: A revenue audit report was retrieved successfully.
```

nitpick: avoid past tense
```suggestion
      description: Revenue audit report successfully retrieved.
```